source/PlayerInfoPanel.cpp

```diff
@@ -468,15 +501,84 @@ bool PlayerInfoPanel::Scroll(double dx, double dy)
 
 
 
+bool CompareName(const shared_ptr<Ship> &lhs, const shared_ptr<Ship> &rhs) {
```

Comment:
Move these unscoped definitions to the anonymous namespace, or if they should be exported to other files, to a new file and explicitly included in this one.